### PR TITLE
Add default build script

### LINKED ISSUE #17: Add basic build script file
The `.build-script` file is required for a successful deployment so we should add that out of the box.

### PR DESCRIPTION
**Pull request by roborourke**
Fixes #17

### CODE REVIEW

File: .build-script [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#====================#
+# Altis Build Script #
+#====================#
+
+# Install composer dependencies
+composer install --no-dev
+
+# Edit below this line to add any additional build steps.
+#
+# Available binaries:
```
Comment (rmccue) [inline .build-script]: Can we link to the docs for something more evergreen?

Comment (roborourke) [reply .build-script]: Oh yeah good idea!

Comment (roborourke) [reply .build-script]: so useful having the site live :)

**Review by rmccue — APPROVED** [review]

**Comment by joehoyle** [comment]
Hmm I'd prefer to move this default to the infra, thinking we can then always have Altis in control of the build script (the infra would default to `vendor/altis/cloud/.build-script` or something. _That_ file could then call out to a user one, but this will also set us run Altis things at the build step too.

I would say "well this is better than nothing", but if this is for 2.0, then I'd say we just shoot directly for infra level changes.

**Comment by roborourke** [comment]
@joehoyle that's a good idea though think we'll still want this one, I guess with the composer install step removed. There are some docs related to this too so I think lets create a tracking issue on the meta repo.

**Pull request merged**